db/data/scraped_prerequisites.yml

```diff
@@ -9139,16 +9146,16 @@ prerequisites:
         subject_needed_name: LOGICA Y COMPUTACION
         needs: exam
   - type: logical
-    logical_operator: not
+    logical_operator: or
     operands:
     - type: subject
-      needs: enrollment
-      subject_needed_code: '1013'
-      subject_needed_name: LOGICA MODALIDAD AL REVES
-  - type: subject
-    needs: course
-    subject_needed_code: '1023'
-    subject_needed_name: MATEMATICA DISCRETA 1
+      subject_needed_code: '1023'
+      subject_needed_name: MATEMATICA DISCRETA 1
+      needs: course
+    - type: subject
+      subject_needed_code: SRN14
+      subject_needed_name: SRN14 - MATEMÁTICA DISCRETA I
```

Comment:
Is it correct that the code shows up in the subject name? Did we break the scrapper?

Reply:
afaict that's a pre-existing issue: https://github.com/cedarcode/mi_carrera/blob/ad0ae618cb6ccc6661a1e499ddafeb919a6363a1/db/data/scraped_prerequisites.yml#L9770

and will be fixed in the scraper refactor PR: https://github.com/cedarcode/mi_carrera/pull/235/files#diff-0a345fa99c9566f09a88e31e8abda878c16cfd362f5456548c60edc20c498997R9770

Reply:
Good! Interesting 🤔